### ISSUE TITLE
[Item] Shadowmourne deals inconsistent damage across classes

### ISSUE THREAD

**Issue by Chenocide** [dalaranwow/dalaran-wow#5606]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The Chaos Bane explosion from Shadowmourne deals inconsistent damage, depending on which class is using it. Currently retribution paladins seems to benefit the most from it, with unholy death knights being second and warriors last. Currently Chaos Bane seems to benefit from spell power, which falls in line with the observed result, that paladins and then death knights benefit the most from it. 
Apparently Chaos Bane shouldn't be able to to critical strikes but the observed result is that all 3 classes have a lot of critical strikes with Chaos Bane. 

**Expected behaviour**: Tell us what should happen instead
Shadowmourne should perform at the same level for all 3 classes, although some classes might have a difrerent approach as how to benefit from Chaos Bane and the strength buff. According to some sources, Chaos Bane almost never crits. Chaos Bane is supposed to split it's fixed damage across all targets within range, as seen on pictures provided later on, it can sometimes reach 10k total damage on 1 proc, with a retribution paladin.

**Steps to reproduce the problem**:

1. Acquire Shadowmourne for all 3 classes
2. Apply raid buffs, mostly spell power buffs or drop gear
3. Log Chaos Bane performance across all 3 classes

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

Here's Chaos Bane from all 3 classes, from the same raid.
![](https://puu.sh/ByyBB/254101d5f4.png)
![](https://puu.sh/ByyC2/da0704854c.png)
![](https://puu.sh/ByyCv/a7d10720b5.png)

Proof that Chaos Bane crits on Dalaran-wow.
![](https://puu.sh/Byzni/53a7107664.png)

Chaos Bane damage on dummy, with and without gear (less spell power).
![](https://puu.sh/Byzjg/8ea19b7937.png)
![](https://puu.sh/Byzjw/9e583fc27a.png)

http://wowwiki.wikia.com/wiki/Shadowmourne

"Chaos Bane cannot crit under normal circumstances. Retribution Paladins have been experiencing rare occurrences of Chaos Bane crits on raid mobs, which is believed to be caused by the talent Sanctity of Battle. Chaos Bane can also crit if the target is much lower level than the player."

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

Chaos Bane ID: 71904

**Include Screenshots from the issue if necessary**

**Comment by daxpedda** [comment]
I'm going to summarize the issues here and point out some mistakes:
1. [Chaos Bane](https://wotlk.evowow.com/?spell=71904) scales with spell power, we don't know if it should ([Retail WoL](http://www.worldoflogs.com/reports/0wu7ij2fqswbq4sz/xe/?s=5991&e=6596&x=spellID+%3D+71904) shows inconsistencies in damage, it has to scale with something).
2. Chaos Bane scales with melee/spell crit chance, it shouldn't ([WoWWiki](http://wowwiki.wikia.com/wiki/Shadowmourne#Mechanics)) except from abilities like [Sanctity of battle](https://wotlk.evowow.com/?spell=35397), other abilities aren't recorded anywhere else.

I can't find exact data on how it scales, maybe it scales of off some of the spell power, maybe only from % spell damge (though that wouldn't explain all inconsistencies on WoL either) and it doesn't even seem like [Soul Fragment](https://wotlk.evowow.com/?spell=71905) has a 100% proc chance.

**Comment by Chenocide** [comment]
I might add that it's with 99% certainty the same issue with Bryntroll. 

ID's:

50709 - HC
50415 - Normal

I haven't searched for any info on this but it should most likely only scale with +% damage talents.

**Comment by Lothloryen** [comment]
Thank you very much for the report, this has been fixed and will be deployed to the live realm with the next maintenance.

**Comment by Chenocide** [comment]
The latest fix to Shadowmourne seems to have nerfed it more than what would be blizzlike. 
Retribution paladins are supposed to be able to rarely crit with Chaos Bane.

http://wowwiki.wikia.com/wiki/Shadowmourne
"Chaos Bane cannot crit under normal circumstances. Retribution Paladins have been experiencing rare occurrences of Chaos Bane crits on raid mobs, which is believed to be caused by the talent Sanctity of Battle. Chaos Bane can also crit if the target is much lower level than the player."

Currently retribution paladins have no crits with Shadowmourne. 
![](https://i.imgur.com/EEW0Qcm.png)

I don't know at what rate Chaos Bane should crit for ret paladins, but I would expect it to happen at least once during a 4 hour long raid.
